http2/README.md

```diff
@@ -11,107 +11,80 @@ Note: due to the lack of H2C support in popular frameworks, the Ruby and Python
 
 ## Downloading the Samples
 
-Clone this repo and go to the app you want to test
+Clone this repo 
 
 ```shell
 # Skip this if you cloned the repository already
 > git clone https://github.com/SAP-samples/cf-routing-samples
 > cd cf-routing-samples/http2
 
-# Depending on the app you want to try, go into that respective directory, e.g. go-http2
-> cd go-http2
 ```
-
-### How to deploy H2C apps
-
-Make sure you have [CF CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) version 8 or higher installed
-
+## Deploying all the apps at once
+### Pre-requisites
+Set the desired domain in [vars.yml](vars.yml) file:
```

Comment:
Maybe we can refer users to use the `cf --var domain=the.domain.example.com` syntax instead of adding a vars.yml file? This applies to all mentions of `--vars-file vars.yml` throughout the other README.md files.

Then we can maybe even remove the vars.yml altogether?